addons/opacity-slider/addon.json

```diff
@@ -2,6 +2,13 @@
   "name": "Opacity slider",
   "description": "Adds an opacity slider to the costume editor color picker.",
   "tags": ["editor", "costumeEditor", "beta", "featured"],
+  "info": [
+    {
+      "type": "info",
+      "text": "This doesn't work on bitmap mode.",
```

Comment:
I don't think this is needed as a separate info; "vector costume editor" probably works.

Reply:
I disagree - `vector costume editor color picker` is too long.